deploy/setup-project.sh

```diff
@@ -1,6 +1,17 @@
 #!/bin/bash
 
+set -o nounset
+set -o errexit
+
 IAM_NAME="$GCEPD_SA_NAME@$PROJECT.iam.gserviceaccount.com"
+
+# Cleanup old Service Account and Key
+rm -f $SA_FILE
```

Comment:
One more recommendation I found was to put quotes around all variables, to handle things like spaces in filenames

Reply:
done